### ISSUE TITLE
Online training - problem definition

### ISSUE THREAD

**Issue by gnperdue** [fermilab-accelerator-ai/meetings#12]
Need to define what this looks like - how do we collect and apply gradients? how expensive is it to reflash FPGAs? is it possible to store and update network weights in a way that doesn't require reflashing? where does the compute agent operate that drives the training loop and computes gradients?
